### DISCUSSION TITLE
Kafka 3.5.0 version

### DISCUSSION THREAD

**Discussion by ulpcan** [strimzi/strimzi-kafka-operator#8859] [General]
Hello there,

We are waiting next release of strimzi kafka operator for Apache Kafka 3.5.0 before going prod

Do you have any plan of calendar for the next release?

Thanks!

**Comment by scholzj** [comment]
As discussed on the last weeks' community call - we are waiting for some last system test fixes before doing the first release candidate of Strimzi 0.36.0. I expect it to happen this week.
